server/src/yml-objects/AbstractYmlObject.ts

```diff
@@ -32,6 +32,10 @@ export abstract class AbstractYmlObject implements CompletionItem {
     public data?: any;
     /* End of overriden properties. */
 
+    private sourceElementName: string;
+    private documentationAsText: string;
+    public kindName: string = 'Object';
```

Comment:
Not sure what this is either.

Reply:
You're right. This misses documentation.